src/screens/documentDetails/components/documentActivity/index.tsx

```diff
@@ -0,0 +1,48 @@
+import Grid from "@mui/material/Grid";
+import {Button, Card, CardContent, Chip, Stack, Typography} from "@mui/material";
+import {FormattedDate} from "../../../../components/formatedDate";
+import React from "react";
+import {DocumentDto} from "../../../../services/api/dtos";
+
+interface DocumentActivityProps {
+    document: DocumentDto;
+}
+
+function DocumentActivity({document}: DocumentActivityProps) {
+
+    const assignedUser = (document.assignedUser?.surname && document.assignedUser?.name)
+        ? `${document.assignedUser?.surname} ${document.assignedUser?.name}`
```

Comment:
optional chaining after validating existence of properties seems redundant

Reply:
true, autocomplete went brr